src/js/index.js

```diff
@@ -1 +1,16 @@
 import "../styles/index.css";
+import { startingPage } from "./starting-page";
+import { firstChapter, secondChapter, thirdChapter } from "./demo-story";
+import { render } from "./render";
+import { gameLogic } from "./logic";
+import { Story } from "./stories-factory";
```

Comment:
It's a very minor thing xD I prefer story-factory instead of stories-factory. The factory produces one story when invoked not multiple stories. Also chapter-factory is already written in singular wording.

Reply:
Changed in stories-factory branch.